kolibri/core/analytics/tasks.py

```diff
@@ -70,3 +71,11 @@ def schedule_ping(
             pass
     elif conf.OPTIONS["Deployment"]["DISABLE_PING"]:
         job_storage.clear(job_id=DEFAULT_PING_JOB_ID)
+
+
+def trigger_pingback():
+    try:
+        job = job_storage.get_job(job_id=DEFAULT_PING_JOB_ID)
+        job.retry_in(timedelta(seconds=1))
```

Comment:
Unfortunately, this won't work - I implemented the `retry_in` method for the purpose of allowing jobs to dynamically define their retry interval (so that they could implement exponential backoff schemes, for example) - but the pingback is already scheduled with a constant retry_interval, so this will fail, because of this check:

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/tasks/storage.py#L457